### PR TITLE
v84.3.0 Release Candidate.

### PR DESCRIPTION
**Pull request by galiak11**
# 84.3.0

This minor release adds new theming extensions for Lists and minor bug fixes and documentation updates to multiple components.

## New features

Theming extensions are now available for Lists:

```swift
let cell = MDCSelfSizingStereoCell()
let scheme = MDCContainerScheme()

cell.applyTheme(withScheme: scheme)
```

#### MDCChipView

*modified* class: `MDCChipView`

*new* property: `mdc_legacyFontScaling` in `MDCChipView`

### List+Theming

**New extension.**

### Buttons

* [Add snapshot test for theming extensions (#7390)](https://github.com/material-components/material-components-ios/commit/62e747ea094681e779ca4b294c25637635a9101d) (Cody Weaver)
* [Remove dependency on TypographyThemer in theming extension.  (#7391)](https://github.com/material-components/material-components-ios/commit/daee4c058c2d52487689c56cf0822cbd5073a2d6) (Cody Weaver)
* [Update titleFont:forState: API (#7402)](https://github.com/material-components/material-components-ios/commit/80037e4684ee65c55a38fa34161ae7aaf9630ec3) (Cody Weaver)
* [[Ripple] fix button to update ripple states correctly (#7337)](https://github.com/material-components/material-components-ios/commit/7c481c023ef2fa01e8ab0824a58e3ae6bc648303) (Galia Kaufman)
* [Update outline theming extension (#7403)](https://github.com/material-components/material-components-ios/commit/936d04ab3589ff6747798df6607c3d8d288edec0) (Cody Weaver)

### Chips

* [Add M2 dynamic type support to Chips. (#7394)](https://github.com/material-components/material-components-ios/commit/2744b6c04817d2bb104d1d368bcc7fc92625d92e) (Wenyu Zhang)
* [refactor chip title font. (#7406)](https://github.com/material-components/material-components-ios/commit/de3c751ba8034fd22371661bab058ff2b90c7fac) (Wenyu Zhang)

### FlexibleHeader

* [Clarify the docs for preferredStatusBarStyle. (#7415)](https://github.com/material-components/material-components-ios/commit/4f7145da2bf8b9cb1f6d854f67bf055dce00d1ae) (featherless)

### List

* [Add List theming extension (#7397)](https://github.com/material-components/material-components-ios/commit/36018edb6101c912edb59ca27128ef2e31bb72f1) (Andrew Overton)

### NavigationDrawer

* [Explain use of AppBar elevation (#7399)](https://github.com/material-components/material-components-ios/commit/a197368ffd3c7f402ddd76784922193576a1f347) (Robert Moore)

### ProgressView

* [Add cornerRadius snapshot tests. (#7416)](https://github.com/material-components/material-components-ios/commit/9664fce4b3cd3bec07c50d8e7a7a7720ac003332) (featherless)

## Multi-component changes

* [Roll-forward a shadow elevation issue. (#7396)](https://github.com/material-components/material-components-ios/commit/ab5e6f69c91ebb9d7cc0092d8641d585cfce1bee) (Yarden Eitan)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
+# 84.3.0
+
+This minor release adds dynamic type support to Chips, new theming extensions for Lists, updates to Bottom Drawer elevation shadow. It also includes minior bug fixes and documentation updates to multiple components.
```
Comment (jverkoey) [inline CHANGELOG.md:3]: To clarify: we didn't add Dynamic Type support in general, we added support for the new Material Dynamic Type curves.

"This minor release adds support for new Material Dynamic Type curves to Chips"

Please clarify this below as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+
+## New features
+
+# Dynamic type support
+
+'MDCChipView' now support Dynamic font type.
```
Comment (jverkoey) [inline CHANGELOG.md:9]: Replace the `'` with a back-tick (`) in order to make it appear as code.

File: CHANGELOG.md (hunk 1 above)
Comment (galiak11) [reply CHANGELOG.md:3]: I believe this API has a bug and will be replaced in the next release, which is why we skipped this feature in this release.

File: CHANGELOG.md (hunk 2 above)
Comment (galiak11) [reply CHANGELOG.md:9]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,63 @@
+# 84.3.0
+
+This minor release adds new theming extensions for Listsz and minior bug fixes and documentation updates to multiple components.
+
+## New features
+
+# Dynamic type support
```
Comment (romoore) [inline CHANGELOG.md:7]: We already had support for Dynamic Type, but the difference is now we're supporting Dynamic Type using MDCFontScaler.  I think we should be careful in this wording so it doesn't sound like we didn't have any support previously.

Comment (romoore) [reply CHANGELOG.md:7]: Done

**Review by romoore — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,71 @@
...
+
+*modified* class: `MDCChipView`
+
+*new* property: `mdc_legacyFontScaling` in `MDCChipView`
+
+### List+Theming
+
+**New extension.**
+
+## Component changes
```
Comment (romoore) [inline CHANGELOG.md:30]: Please remove "Component changes" since the section is empty.

**Review by romoore — APPROVED** [review]
LGTM, but please address comment  in CHANGELOG before merging.

**Review by romoore — CHANGES_REQUESTED** [review]
Please merge the `stable` branch into `release-candidate` so that importing to Google will include the latest changes from `stable`.

**Review by romoore — APPROVED** [review]
Please clean-up CHANGELOG. LGTM otherwise! :D

File: CHANGELOG.md (hunk 4 above)
Comment (galiak11) [reply CHANGELOG.md:30]: Done

**Pull request merged**